### ISSUE TITLE
about the data imgsize

### ISSUE THREAD

**Issue by dzz416** [IanYeung/RealVSR#3]
Sorry, I don’t understand very well. Why are the LR and HR images the same size? If I want to use your data to train a super-resolution network, I should enter a small LR and a large HR. How can I get a small LR in the real world?

**Comment by IanYeung** [comment]
> Sorry, I don’t understand very well. Why are the LR and HR images the same size? If I want to use your data to train a super-resolution network, I should enter a small LR and a large HR. How can I get a small LR in the real world?

Similar dataset construction methods could be adopted. But you may need to modify the registration algorithm. Or you can try directly downsample the current LR.

**Comment by dzz416** [comment]
> > Sorry, I don’t understand very well. Why are the LR and HR images the same size? If I want to use your data to train a super-resolution network, I should enter a small LR and a large HR. How can I get a small LR in the real world?
> 
> Similar dataset construction methods could be adopted. But you may need to modify the registration algorithm. Or you can try directly downsample the current LR.

Logically speaking, the size of the LR obtained by shooting should be small. Did you upsample it or downsample the HR? Why do you do that?

**Comment by IanYeung** [comment]
> > > Sorry, I don’t understand very well. Why are the LR and HR images the same size? If I want to use your data to train a super-resolution network, I should enter a small LR and a large HR. How can I get a small LR in the real world?
> > 
> > 
> > Similar dataset construction methods could be adopted. But you may need to modify the registration algorithm. Or you can try directly downsample the current LR.
> 
> Logically speaking, the size of the LR obtained by shooting should be small. Did you upsample it or downsample the HR? Why do you do that?

Yes, traditionally super-resolution will increase pixels, in our case, maybe it is more closer to "digital zooming". The LR and HR have the same size because we adopt the registration algorithm in RealSR.

**Comment by dzz416** [comment]
> > > > Sorry, I don’t understand very well. Why are the LR and HR images the same size? If I want to use your data to train a super-resolution network, I should enter a small LR and a large HR. How can I get a small LR in the real world?
> > > 
> > > 
> > > Similar dataset construction methods could be adopted. But you may need to modify the registration algorithm. Or you can try directly downsample the current LR.
> > 
> > 
> > Logically speaking, the size of the LR obtained by shooting should be small. Did you upsample it or downsample the HR? Why do you do that?
> 
> Yes, traditionally super-resolution will increase pixels, in our case, maybe it is more closer to "digital zooming". The LR and HR have the same size because we adopt the registration algorithm in RealSR.

If registration is not performed, shouldn’t it be closer to the actual situation if the part corresponding to the HR in the captured LR image (large field of view) is directly cropped as an LR image? Do you still have the original data?

**Comment by IanYeung** [comment]
> > > > > Sorry, I don’t understand very well. Why are the LR and HR images the same size? If I want to use your data to train a super-resolution network, I should enter a small LR and a large HR. How can I get a small LR in the real world?
> > > > 
> > > > 
> > > > Similar dataset construction methods could be adopted. But you may need to modify the registration algorithm. Or you can try directly downsample the current LR.
> > > 
> > > 
> > > Logically speaking, the size of the LR obtained by shooting should be small. Did you upsample it or downsample the HR? Why do you do that?
> > 
> > 
> > Yes, traditionally super-resolution will increase pixels, in our case, maybe it is more closer to "digital zooming". The LR and HR have the same size because we adopt the registration algorithm in RealSR.
> 
> If registration is not performed, shouldn’t it be closer to the actual situation if the part corresponding to the HR in the captured LR image (large field of view) is directly cropped as an LR image? Do you still have the original data?

Maybe. But with more serious misalignment problem. Yes I have the original data and maybe I will provide them later.

**Comment by dzz416** [comment]
Ok,thank you for reply
